akka-actor-testkit-typed/src/main/mima-filters/2.6.18.backwards.excludes/product-iterator-type-signature-with-scala-2.13.7.excludes

```diff
@@ -0,0 +1,4 @@
+# Scala 2.13.7 generates different binary when using type wildcards, see https://github.com/scala/bug/issues/12488
+ProblemFilters.exclude[IncompatibleSignatureProblem]("akka.actor.testkit.typed.Effect#SpawnedAnonymousAdapter.productIterator")
+ProblemFilters.exclude[IncompatibleSignatureProblem]("akka.actor.testkit.typed.scaladsl.ScalaTestWithActorTestKit.theSameElementsAs")
+ProblemFilters.exclude[IncompatibleSignatureProblem]("akka.actor.testkit.typed.scaladsl.ScalaTestWithActorTestKit.theSameElementsInOrderAs")
```

Comment:
See https://github.com/scala/bug/issues/12488
The  abstract `ScalaTestWithActorTestKit` is getting it indirectly from a trait it extends from ScalaTest.

Reply:
No worse than users mixing in `Matchers` or where it comes from directly in a test though, right? (So only a problem if a specific Java test did have an override with the old return type, highly unlikely)